Guided Workouts

d-x-s opened the pull request:
Changelog:
- **take package-lock.json from main branch**
- **voice control**
- Guided workout components
- New dependecies (carousel and MUI)
- Navbar styling compatability and bug fixes
- Remove emoji in toasts
- Refactor styles to theme file
- Animated number dial (Spring)

jakji93 commented:
![image](https://github.com/jakji93/airon-fitness/assets/54963621/cceef5c4-6f2f-473b-8165-4d5531091347)
Maybe there is a latest update that wasn't pushed, but set remaining can go less than 0 unless it is intended

jakji93 commented on client/src/components/Workout/GuidedExercise.jsx:
```diff
@@ -0,0 +1,257 @@
+              variant="outlined"
+              sx={styles.workoutSelectButton}
+            >
+              RESET REST
```
reset doesn't do anything on click yet

jakji93 commented on client/src/components/Workout/GuidedExercise.jsx:
```diff
@@ -0,0 +1,257 @@
+              variant="outlined"
+              sx={styles.workoutSelectButton}
+            >
+              +15 SECONDS
```
same as +15, doesn't do anything right now

d-x-s commented:
UI/UX Improvements:

- Finishing a set will reset the timer and start it if it is not already started
- Fixed hover styles for timer button
- Now a clear color for timer ON and timer OFF
- Fixed exercise table header colors
- Timer immediately starts counting down after click
